bin/sync

```diff
@@ -0,0 +1,9 @@
+#! /usr/bin/env ruby
+# frozen_string_literal: true
+
+# rubocop:disable Packaging/RequireRelativeHardcodingLib
+require_relative "../lib/trmnl/i18n/editor"
+# rubocop:enable Packaging/RequireRelativeHardcodingLib
```

Comment:
@bkuhlmann I couldn't find a way around this. The autocorrected solution did not work.